src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,24 +1,14 @@
 databaseChangeLog:
-  - changeSet:
-      id: 1
-      author: sa
-      changes:
-        - createSequence:
-            sequenceName: id_seq
-            incrementBy: 1
-            startValue: 1
             
   - changeSet:
-      id: 2
+      id: "create sequence and table for process type"
```

Comment:
Okay. So, I'll make changes in the primary keys for the tables. Remove auto increment and use UUIDS. However, there will be a problem in loadData because of this as it would be impossible to know which process_type_id to enter in the action_requests.csv file as the foreign key.